README.md

```diff
@@ -115,6 +115,7 @@ The `CommonFactory` reads a message's router configuration from the `mq.json` fi
     * filters - pin's message's filters
         * metadata - a metadata filters
         * message - a message's fields filters
+    * messageRecursionLimit - an integer number denotes how deep nested protobuf message might be, default = 100
```

Comment:
This incorrect place, because common reads messageRecursionLimit from mq_router.json